### ISSUE TITLE
GTTweaker check

### ISSUE THREAD

**Issue by Dream-Master** [GTNewHorizons/NewHorizons#199]
@Namikon Can you check may code on the GTTweaker because of the Logistic Issue and The new Buildcraft 7 Issue. Maybe i did something wrong in the code. Thanks

**Comment by Namikon** [comment]
Your mod is not the issue here. A plain instance with just buildcraft and minetweaker has the same problem

**Comment by Namikon** [comment]
Hm maybe not, the space between the text seems to be a format-"issue" from Minetweaker, to improve readability. After I removed all the unknown items it reloads the script without any issue. Are you sure the item names haven't changed and that's the reason why you get the "cannot find xx" messages?
It seems the spacing is not the problem.
I noticed that a lot of Buildcraft items are missing in BC7, that's why your current script fails
